From ad6e2ddaf8421e992fea2d2e77c491255d6a035b Mon Sep 17 00:00:00 2001
From: Remi Collet <remi@remirepo.net>
Date: Fri, 17 Jun 2022 14:00:36 +0200
Subject: [PATCH] Fix: Deprecated: Creation of dynamic property (8.2)

---
 tests/bug_62354.phpt | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/tests/bug_62354.phpt b/tests/bug_62354.phpt
index d2a173b3..6330476c 100644
--- a/tests/bug_62354.phpt
+++ b/tests/bug_62354.phpt
@@ -4,7 +4,9 @@ Constructing AMQPQueue with AMQPConnection segfaults
 <?php if (!extension_loaded("amqp")) print "skip"; ?>
 --FILE--
 <?php
-class Amqptest {};
+class Amqptest {
+	public $conn = NULL;
+};
 $o = new Amqptest();
 $o->conn = new AMQPConnection();
 $funcs = array(
@@ -20,4 +22,4 @@ getLogin => 'guest'
 getPassword => 'guest'
 getPort => 5672
 getVHost => '/'
-isConnected => false
\ No newline at end of file
+isConnected => false
